### PR TITLE
Add time_card handler

### PR DESCRIPTION
**Pull request by segiddins**
Closes https://github.com/segiddins/time-card/issues/2 ?
@indirect

### CODE REVIEW

File: lita_config.rb [hunk 1]
```diff
@@ -31,4 +31,6 @@
     config.handlers.tweet.consumer_key = ENV.fetch("TWITTER_CONSUMER_KEY")
     config.handlers.tweet.consumer_secret = ENV.fetch("TWITTER_CONSUMER_SECRET")
   end
+
+  config.handlers.time_car.token = ENV["TIME_CARD_PASSWORD"]
```
Comment (indirect) [inline lita_config.rb]: time_car?

File: lib/lita/handlers/time_card.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+module Lita
+  module Handlers
+    class TimeCard < Lita::Handler
+      namespace :time_card
+      config :token
+
+      route %r{^time_card (\d+)(?: (\d{4}-\d{2}-\d{2}))? (.+)}m,
+            :log_time,
+            command: true,
+            help: { "time_card MINUTES [DATE] MESSAGE" => "Add a time card entry." }
```
Comment (indirect) [inline lib/lita/handlers/time_card.rb:18]: shouldn't this also accept e.g. `1h`, `2.5h`?

**Review by indirect — CHANGES_REQUESTED** [review]

File: lib/lita/handlers/time_card.rb (hunk 2 above)
Comment (segiddins) [reply lib/lita/handlers/time_card.rb:18]: I guess it could? default to minutes when no suffix given?

**Comment by segiddins** [comment]
@indirect I believe this should address your concerns?

**Review by indirect — APPROVED** [review]

**Comment by indirect** [comment]
Thanks!

**Pull request merged**